frontend/src/app/pages/exercises/exercise/shared/simulated-region-overview/tabs/behavior-tab/behaviors/report/simulated-region-overview-behavior-report.component.html

```diff
@@ -0,0 +1,174 @@
+                        name="reportType"
+                        id="once"
+                        #once
+                        autocomplete="off"
+                        checked
+                    />
+                    <input
+                        type="radio"
+                        class="btn-check"
+                        name="reportType"
+                        id="repeating"
+                        #repeating
+                        autocomplete="off"
+                    />
+                </div>
+                <div class="col-6">
+                    <div class="input-group input-group-sm float-start">
+                        <input
+                            #timeInput
+                            type="number"
+                            min="0"
+                            step="0.1"
+                            value="5"
+                            [disabled]="once.checked"
+                            class="form-control form-control-sm d-inline-block no-validation"
+                        />
+                        <span class="input-group-text">Min</span>
+                    </div>
+                </div>
+            </div>
+
+            <div class="row align-items-center">
+                <div class="col-5 mt-4">
+                    <button
+                        type="button"
+                        class="btn btn-primary"
+                        [disabled]="
+                            selectedInformation === 'noSelect' ||
+                            (repeating.checked &&
+                                activities[selectedInformation] !== undefined)
+                        "
+                        (click)="
+                            createReports(
+                                selectedInformation,
+                                timeInput.value,
+                                repeating.checked
+                            )
+                        "
+                    >
+                        Erstelle Bericht
+                    </button>
+                </div>
+                <div class="col-7">
+                    <div
+                        class="ms-3"
+                        *ngIf="
+                            selectedInformation !== 'noSelect' &&
+                            repeating.checked &&
+                            activities[selectedInformation] !== undefined
+                        "
+                    >
+                        <i
+                            class="bi bi-exclamation-triangle-fill text-warning"
+                        ></i>
+                        Es werden bereits regelmäßig Berichte für diese
+                        Information erstellt.
+                    </div>
+                </div>
+            </div>
+        </div>
+
+        <h5>Regelmäßige Berichte</h5>
+
+        <ng-container *ngFor="let information of reportableInformations">
+            <div *ngIf="reportBehaviorState?.activityIds?.[information]">
```

Comment:
Note to self: These two lines might be mergeable

Reply:
They are not, because we would need to check whether the value is undefined or not.